core/controllers/suggestion_test.py

```diff
@@ -312,6 +518,15 @@ def test_accept_suggestion(self):
             suggestion_models.STATUS_ACCEPTED)
         self.logout()
 
+    def test_suggestion_list_handler_with_invalid_query_field(self):
```

Comment:
Try the query with a combination of valid and invalid fields.

Reply:
There are already a lot of tests with valid queries like test_query_question_suggestions, test_suggestion_list_handler etc